config/application.rb

```diff
@@ -11,6 +16,13 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
 
+    # Serve out any static files found in Rails.application.secrets.import_dir
+    # This is required for CSV imports that load files from disk.
+    # If import dir is not defined, default to the local spec fixtures directory
+    config.serve_static_files = true
+    import_dir = Rails.application.secrets.import_dir || Rails.root.join('spec', 'fixtures').to_s
+    config.middleware.use ::ActionDispatch::Static, import_dir
```

Comment:
So does this mean Rails serves everything in the specified directory?  For production that will be a directory containing multiple mounted drives containing thousands of files.  Will that perform OK?